Tools/TestWebKitAPI/Tests/WebKitGLib/TestInputMethodContext.cpp

```diff
@@ -281,7 +281,7 @@ class InputMethodTest: public WebViewTest {
 
     ~InputMethodTest()
     {
-        webkit_user_content_manager_unregister_script_message_handler(m_userContentManager.get(), "imEvent");
+        webkit_user_content_manager_unregister_script_message_handler(m_userContentManager.get(), "imEvent", "");
```

Comment:
Are you sure this is correct? I thought using an empty string as the world name produced a new script world that is distinct from the default script world. Shouldn't you pass `NULL` here instead?

Reply:
Right NULL should be passed here, thank you for pointing this out

Reply:
(Oops, I really meant `nullptr`. We don't use `NULL` anywhere in C++ code anymore.)